### PR TITLE
Change dep back to package:test

### PR DESCRIPTION
**Pull request by dnfield**
The original reason for this doesn't make sense - the tool still depends on package:test. This is holding up an internal roll of this package.

@natebosch @jonahwilliams @jamesderlin

### CODE REVIEW

File: packages/file/test/common_tests.dart [hunk 1]
```diff
@@ -2225,6 +2225,7 @@ void runCommonTests(
 
           setUp(() {
             f = fs.file(ns('/foo'));
+            f.createSync(recursive: true);
```
Comment (natebosch) [inline packages/file/test/common_tests.dart:2228]: How is this related? Can you mention it in the commit message?

File: packages/file_testing/pubspec.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+version: 2.2.0
 authors:
 - Todd Volkert <tvolkert@google.com>
 description: Testing utilities for package:file
 homepage: https://github.com/google/file.dart/packages/file_testing
 
 dependencies:
   meta: ^1.1.2
-  test_api: ^0.2.2
+  test: ">=1.0.0 <2.0.0"
```
Comment (natebosch) [inline packages/file_testing/pubspec.yaml:10]: The equivalent and simpler format is:

```suggestion
  test: ^1.0.0
```

File: packages/file/test/common_tests.dart (hunk 1 above)
Comment (dnfield) [reply packages/file/test/common_tests.dart:2228]: Fixing CI. This is unrelated, this was an existing broken test.

**Comment by dnfield** [comment]
I do not know why CI is failing - it's not making much sense to me and not failing locally, and seems to have been failing on recent PRs as well. Going to land this to unblock the internal roll.
